### DISCUSSION TITLE
Options for local CRUD only?

### DISCUSSION THREAD

**Discussion by har-nick** [dropbox/Store#556] [Q&A] [answered]
Hi! My app doesn't have write access to the network backend. It just makes requests to source data, before writing it to a local database.

While I can make requests well enough, I can't find out how to write to the database without turning the Store into a Mutable one, which requires an Updater. Is this possible at all?

Thanks!

**Comment by matt-ramotar** [comment, marked as answer]
Hey there - If the writes are always dependent on reads, you can provide a `SourceofTruth` to `StoreBuilder`. If writes can be independent of reads, then you need a `MutableStore`.  Although we aren't making local writes only possible through `MutableStoreBuilder`, you can achieve it by providing a no-op `Updater` without a `Bookkeeper`. Completely open to supporting this. Would be happy to review a PR if you want to take a stab at #536
